packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -606,6 +606,15 @@ export class Project extends ProjectBase {
     return new Import(scope, id);
   }
 
+  public static serializeEnvVariables(environmentVariables: { [name: string]: BuildEnvironmentVariable }):
```

Comment:
Public docstring if you want to expose this.

Reply:
Done.